pom.xml

```diff
@@ -33,8 +33,13 @@
     </developers>
 
     <properties>
-        <maven.compiler.source>1.7</maven.compiler.source>
-        <maven.compiler.target>1.7</maven.compiler.target>
+        <junit5.version>5.5.2</junit5.version>
+        <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
+
+        <!-- Test sources at 1.8 to allow using JUnit5 -->
+        <java.version>1.7</java.version>
```

Comment:
Can we update the java version to a more recent version?

Reply:
No, we need to support Android